.baseline/checkstyle/checkstyle.xml

```diff
@@ -423,5 +423,11 @@
         </module>
 
         <!-- Stricter checks end -->
+
+        <module name="RegexpSinglelineJava">
```

Comment:
would you be opposed to contributing this to baseline? internally, in most cases you want this to be a managed thing rather than registering a shutdown hook

Reply:
here it goes: https://github.com/palantir/gradle-baseline/pull/473